CHANGELOG.rst

```diff
@@ -14,6 +14,13 @@ Change Log
 Unreleased
 **********
 
+[3.9.5] - 2023-02-24
+********************
+
+Fixed
+=====
+* load all signals in consume_events command before running the consumer
```

Comment:
It would be nice to explain what the failure mode is that this fixes. Maybe something like this?

```suggestion
* ``consume_events`` command now loads all public signals so that the consumer can load signals by ``event_type`` even if their modules were not already imported
```